packages/client/hmi-client/src/services/project.ts

```diff
@@ -181,17 +184,17 @@ async function home(): Promise<IProject[] | null> {
  * Get the icon associated with an Asset
  */
 const icons = new Map([
-	[ProjectAssetTypes.DOCUMENTS, 'file'],
-	[ProjectAssetTypes.MODELS, 'share-alt'],
-	[ProjectAssetTypes.DATASETS, 'sliders-v'],
-	[ProjectAssetTypes.SIMULATIONS, 'cog'],
-	[ProjectAssetTypes.SIMULATION_RUNS, 'eye'],
-	[ProjectAssetTypes.CODE, 'code'],
-	['overview', 'inbox']
+	[ProjectAssetTypes.DOCUMENTS, 'pi pi-file'],
+	[ProjectAssetTypes.MODELS, 'pi pi-share-alt'],
+	[ProjectAssetTypes.DATASETS, DatasetIcon],
+	[ProjectAssetTypes.SIMULATIONS, 'pi pi-cog'],
+	[ProjectAssetTypes.SIMULATION_RUNS, ResultsIcon],
+	[ProjectAssetTypes.CODE, 'pi pi-code'],
+	['overview', 'layout']
```

Comment:
What I mentioned above, let's drop the _Prime Icons_, just find the ones that looks close enough for now.